CHANGELOG.md

```diff
@@ -28,6 +28,9 @@
 ## Next
 
 * Backwards incompatible change: Rename `DCOS_Docker` backend to `Docker` backend.
+* Replace `generate_config_path` with `build_artifact` that can either be a 
```

Comment:
As above, note that this is a "Backwards incompatible change".

Note that when we do a release to use these changes, we'll have to change existing tests.